install/install.ps1

```diff
@@ -58,12 +58,17 @@ if (!(Test-Path $configDir)) {
 $configFile = Join-Path $configDir "config.json"
 
 if (!(Test-Path $configFile)) {
+    $dummyConfigCreated = $true
     Write-Output '{"auths":{}}' | Out-File $configFile -Encoding ASCII
 }
 
 $configEditPath = [System.IO.Path]::Combine(".", $tempdir, "config-edit.exe")
 &$configEditPath "--helper" "acr-windows" "--config-file" "${configFile}"
 
+if ($dummyConfigCreated) {
+    Remove-Item -Force "${configFile}.bak"

```

Comment:
Can you check if the file exists before trying to remove it? Just so it's cleaner. Same for the bash script.